android/build.gradle

```diff
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 25
```

Comment:
Same thing :)
- 25.0.2 is the lastest
- compileSdkVersion should stay at 23, react-native-linear-gradient doesn't take advantage of features that needs API 24 or 25